git_pw/config.py

```diff
@@ -25,6 +26,9 @@ def _get_config(key):
     except subprocess.CalledProcessError:
         output = ''
 
+    if os.environ.get('PW_%s' % key.upper()) is not None:
```

Comment:
Can do.  Curious, won't we still need code to pull the variable?  Just not sure how to resolve this with the .gitconfig settings.